tests/test_butler.py

```diff
@@ -619,6 +619,13 @@ def setUp(self):
         uri = ButlerURI(config[".datastore.datastore.root"])
         self.bucketName = uri.netloc
 
+        # set up some fake credentials if they do not exist
+        if not os.path.exists("~/.aws/credentials"):
```

Comment:
Is this because sometimes you want to be able to use real credentials in the test?

Reply:
Yes. But I guess if I comment out mock decorators and set the env vars myself is also good enough. Removed.